Course_Part_04/src/Assignment16_IsPrime.java

```diff
@@ -0,0 +1,25 @@
+public class Assignment16_IsPrime {
+
+    public static void main(String[] args) {
+
+        for (int num = 2; num < 101; num++) {
+            if (isPrime(num)) {
+                System.out.printf("The number %d is prime%n", num);
+            } else {
+                System.out.printf("The number %d is not a prime%n", num);
+            }
+        }
+
+    }
+
+    private static boolean isPrime(int num) {
+
+        for (int i = 2; i < num; i++) {
```

Comment:
dit is een prima for loop voor de bepaling of een getal prime is of niet. 

er is nog een snellere manier om dit te bepalen (minder iteraties nodig):

```
	private static boolean isPrimeFast(int number) {

		// https://www.beterrekenen.nl/website/index.php?pag=248
		for (int i = 2; i <= Math.sqrt(number); i++) {
			if (number % i == 0) {
				return false;
			}
		}
		return true;
	}
```